sinks/grpsink/grpsink.go

```diff
@@ -52,13 +61,18 @@ var _ sinks.SpanSink = &GRPCSpanSink{}
 //
 // Any grpc.CallOpts that are provided will be used while first establishing the
 // connection to the target server (in grpc.DialContext()).
-func NewGRPCSpanSink(ctx context.Context, target, name string, log *logrus.Logger, opts ...grpc.DialOption) (*GRPCSpanSink, error) {
+
+func Create(
+	server *veneur.Server, name string, logger *logrus.Entry,
+	config veneur.Config, sinkConfig veneur.SpanSinkConfig, ctx context.Context,
+) (sinks.SpanSink, error) {
+	grpcConfig := sinkConfig.(GRPCSpanSinkConfig)
 	name = "grpc-" + name
```

Comment:
I think we may need that the name remains unchanged for metric routing to work (i.e. remove this line).